tests/ci_install.sh

```diff
@@ -1,9 +1,33 @@
 set -x   # Show which command is being run
 
-echo "Travis build stage: $TRAVIS_BUILD_STAGE_NAME"
+[[ -n ${RUNNER_OS} ]] && export TRAVIS_OS_NAME=${RUNNER_OS}
```

Comment:
Would love to see this file be reduced/eliminated, but not sure if that's feasible...

Reply:
I missed that. It was unnecessary and now it's dropped